File: soal_1/soal1.go
package main

import (
	"fmt"
	"strings"
)

func A000124Formatted(n int) string {
	if n <= 0 {
		return ""
	}

	parts := make([]string, 0, n)
	for k := 0; k < n; k++ {
		val := k*(k+1)/2 + 1
		parts = append(parts, fmt.Sprintf("%d", val))
	}

	return strings.Join(parts, "-")
}

func main() {
	inputs := []int{7, 1, 12}

	for _, n := range inputs {
		fmt.Printf("Input: %d\nOutput: %s\n\n", n, A000124Formatted(n))
	}
}

File: soal1.js
function A000124Formatted(n) {
    if (n <= 0) return "";

    const parts = [];
    for (let k = 0; k < n; k++) {
        const val = (k * (k + 1)) / 2 + 1;
        parts.push(val.toString());
    }

    return parts.join("-");
}

const inputs = [7, 1, 12];

for (const n of inputs) {
    console.log(`Input: ${n}`);
    console.log(`Output: ${A000124Formatted(n)}\n`);
}